src/test/resources/TestNG.xml

```diff
@@ -1,12 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="TestNG">
-
+<suite name="TestNG" parallel = "methods" thread-count="2">
+    <parameter name="BaseURL" value="https://bbb.testpro.io/"/>
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="PlaylistTest"/>
+                <methods>
+                    <include name ="createPlaylistAddSong"/>
+                    <include name ="deletePlaylistTest"/>
+                </methods>
+
+
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226846084-52873952-9929-4fa3-9bf7-1a216cdae75b.png)

HINT

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG" parallel = "methods" thread-count="2">
    <parameter name="BaseURL" value="https://bbb.testpro.io/"/>
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="PlaylistTest">
                <methods>
                    <include name="createPlaylistAddSong"/>
                    <include name="deletePlaylistTest"/>
                </methods>
            </class>
        </classes>
    </test>
</suite>
```